R/grid.R

```diff
@@ -65,13 +65,32 @@ draw_custom_grid <- function(object, view.name){
   grid.args <- remove_field(grid.args, "equilogs")
   at <- list()
   for (side.name in side.names){
-    usr.at <- axis_axTicks(object, as.side(side.name))
-    if (is.null(usr.at))
-      at[[side.name]] <- grid_axTicks(object, as.side(side.name))
-    else
-      at[[side.name]] <- usr.at
+    side.axis <- as.axis(side.name)
+    if(side.axis == "x") {
+      n.side <- grid.args$nx 
+    } else { 
+      n.side <- grid.args$ny
+    }
+    
+    if(is.null(n.side)){
+      usr.at <- axis_axTicks(object, as.side(side.name))
+      if (is.null(usr.at)) {
+        at[[side.name]] <- grid_axTicks(object, as.side(side.name))
+      } else {
+        at[[side.name]] <- usr.at
+      }
+    } else {
+      plot.lims <- lim(object, as.side(side.name))
+      if(is.na(n.side)) {
+        at[[side.name]] <- NULL
+      } else {
+        at[[side.name]] <- seq.int(plot.lims[1], plot.lims[2], length.out = n.side + 1)
```

Comment:
So we never supported nx/ny - which is weird because I've been using that in repgen. Hmmmm apparently it hasn't been doing what I needed it to do? Anyways, I would like to bump up #343 if that's possible. I can work on it, but @jread-usgs can you add a little more detail?